src/H5VL_log_dataset.cpp

```diff
@@ -348,7 +361,6 @@ static herr_t H5VL_log_dataset_write_elements (void *dset,
         }
         dsel = new H5VL_log_selections (dsid);
         H5VL_LOGI_PROFILING_TIMER_STOP (dp->fp, TIMER_H5VL_LOGI_GET_DATASPACE_SELECTION);
-        CHECK_PTR (dsel)
```

Comment:
Should the above call to `dsel = new H5VL_log_selections (dsid)` be converted to a try..catch like elsewhere in these changes since this `CHECK_PTR` is removed?

Reply:
Added check.